docs/user_guide/components/property_package/general/pure/ConstantProperties.rst

```diff
@@ -0,0 +1,58 @@
+Constant Properties
+=================================================
+
+.. contents:: Contents 
+    :depth: 2
+
+Source
+------
+
+Methods for calculating pure component properties independent of temperature from:
+
+Introduction to Chemical Engineering Thermodynamics, 8th Edition
+J.M. Smith, Hendrick Van Ness, Michael Abbott, and Mark Swihart, 2018, McGraw-Hill
+
+Ideal Liquid Molar Heat Capacity (Constant Pressure and Temperature)
```

Comment:
Only constant pressure here (i.e. it is `cp` and not `cv`). The fact that it doesn't vary with T is a separate issue.